placeholderkv.conf

```diff
@@ -1085,15 +1071,15 @@ acllog-max-len 128
 ################################### CLIENTS ####################################
 
 # Set the max number of connected clients at the same time. By default
-# this limit is set to 10000 clients, however if the Redis server is not
+# this limit is set to 10000 clients, however if the server is not
 # able to configure the process file limit to allow for the specified limit
 # the max number of allowed clients is set to the current file limit
-# minus 32 (as Redis reserves a few file descriptors for internal uses).
+# minus 32 (as the server reserves a few file descriptors for internal uses).
 #
-# Once the limit is reached Redis will close all the new connections sending
+# Once the limit is reached the server will close all the new connections sending
 # an error 'max number of clients reached'.
 #
-# IMPORTANT: When Redis Cluster is used, the max number of connections is also
+# IMPORTANT: With cluster-enabled setup, the max number of connections is also
```

Comment:
We may want to decide at some point what we should call this. :grin:  For a long time I didn't understand at all the foreign terms like CME/CMD.

Maybe you're right here, since the config is actually called `cluster-enabled`. Another possibility, if want to postpone this terminology decision, it to just remove "Redis".

```suggestion
# IMPORTANT: When Cluster is used, the max number of connections is also
```

Reply:
Yeah I picked the config name itself, I think CME/CMD is a Amazon ElastiCache coined term and popularized.

Just using `cluster` seems off to me, as it as a mode of running the server (`standalone`/`cluster`). If you're fine with `cluster-enabled` setup. I will stick to it.

Reply:
THat's a good distinction. Cluster is the whole cluster, all of the nodes together.